### DISCUSSION TITLE
20.11.0 isn't showing as available, will it be?

### DISCUSSION THREAD

**Discussion by Pomax** [coreybutler/nvm-windows#1090] [General] [answered]
v20.11.0 got some changes backported that make life a lot easier (like `import.meta.dirname`) but the latest version of 20 that is available through nvm-windows appears to be v20.8.1 - will v20.9.0, v20.10.0, and v20.11.0 become available "soon" or is 20 cut off with only v21.x getting added?

**Comment by coreybutler** [comment, marked as answer]
**tldr; it's available now**

It's available now. If you're referring to the `nvm ls available` command, it cuts off the most recent 20.x.x versions because the chart accounts for the most recent versions (21.x.x), then counts from the prior base version "up" (i.e. 20.x.x and up), limited to 20 total versions.... there were just a lot of releases on the 20.x.x line, so it's cut off. Historically, there haven't been that many releases, so this could use an update... but I probably won't update it since we have a completely different UI for [Runtime](https://github.com/coreybutler/nvm-windows/wiki/Runtime).

**Comment by Pomax** [comment]
Gah, looks like I was on 1.1.9 instead of 1.1.12.

Cheers!
